sharepoint/sharepoint-ps/sharepoint-online/Remove-SPOContentSecurityPolicy.md

```diff
@@ -0,0 +1,74 @@
+---
+external help file:
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/remove-spocontentsecuritypolicy
+applicable: SharePoint Online
+title: Remove-SPOContentSecurityPolicy
+schema: 2.0.0
+author: jaobrie
+ms.author: jaobrie
+manager: ryannak
+ms.reviewer:
+---
+
+# Remove-SPOContentSecurityPolicy
+
+## SYNOPSIS
+
+Adds an entry to the Content Security Policy configuration.
+
+## SYNTAX
+
+### Default
+
+```powershell
+Remove-SPOContentSecurityPolicy [-Url] <String> [-Directive] <String> 
```

Comment:
Why do we want to specify a directive while removing the CSP entry? isn't the URL parameter sufficient?

Reply:
We could change the behaviour to remove entries for all directives for the given URL but this gives finer control. Open to either option.

Reply:
I would stick with simplicity and just put the URL. if they want to change the directive, just remove the current entry and create a new one.

That said: I don't have a strong opinion here so.. up to you.